app/api/routers/breeze_buddy/configurations/handlers.py

```
@@ -0,0 +1,306 @@
+
+        if call_execution_config:
+            logger.info(
+                f"Configuration created successfully: ID={call_execution_config.id}, "
+                f"merchant={config.merchant_id}, template={config.template}"
+            )
+            return call_execution_config
+        else:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST,
+                detail="Failed to create configuration"
+            )
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)